docs/index.html

```diff
@@ -112,25 +112,19 @@
       <!-- part 1 -->
       <div class="flex mt-10 flex-wrap">
         <div class="flex-1 text-xl">
-          <h1 class="text-3xl sm:text-4xl font-black">The Next Generation of Business Messaging</h1>
-          <p>Build deeper customer relationships at scale with a messaging platform based on trust,quality and choice –-
-            <b>RPAChat </b>!</p>
-          <p>Expanding your messaging strategy just got simpler. RPAChat integrates 5-10+ different chat apps including
-            WhatsApp、WeChat etc.</p>
-          <p>RPAChat provides powerful chat apps API that enables you to send, receive and track chat apps effortlessly</p>
-          <p>You can communicate with your customers on whichever channel they love most.</p>
+          <h1 class="text-4xl sm:text-5xl font-black">The Next Generation of Customer Support</h1>
```

Comment:
Find 5 option

Reply:
1. The Next Generation of Conversation
2. The Power of Automated Conversation At Your Fingertips
3. Streamline Your Conversations, Amplify Your Success
4. Say Hello to Smarter Chatting, Say Hello to RPAChat
5. One Platform, Countless Conversations